packages/next/src/server/lib/lru-cache.ts

```diff
@@ -69,7 +69,7 @@ export class LRUCache<T> {
   }
 
   private evictLeastRecentlyUsed(): void {
-    const lruKey = this.cache.keys().next().value
+    const lruKey = this.cache.keys().next().value || ''
```

Comment:
Are we sure this would be correct? Isn't this just TypeScript being annoying that you can't check for existence in a collection because they types aren't compareable?

For example,

```
const collection = ['foo']
// number is not assignable to string but it's fine that it just returns false
collection.includes(5)
```



Guarding the whole block in a typecheck is safer to not introduce runtime changes just because types were updated.

Reply:
It was a `string | undefined` cannot be passed to param value `string` error, but your suggestion makes sense.